index.js

```diff
@@ -62,6 +65,10 @@ Promise.resolve()
       error('Input Error: Cannot run in watch mode when reading from stdin')
     }
 
+    if (argv.stdout && !stdoutIsFree) {
```

Comment:
`argv.stdout` isn't a good `flag` name for this feature and can be misleading if spotted in examples without documentation, I also don't really think it is a good idea in general to introduce a new option/flag for this as the CLI should be focused on relevant PostCSS usage and not become 'bloated' with less relevant flags in the same scope